ce/common/config/rush/pnpm-lock.yaml

```diff
@@ -98,7 +100,7 @@ packages:
     dev: false
 
   /@azure/cognitiveservices-translatortext/1.0.1:
-    resolution: {integrity: sha512-ByOcyqLKFlpgHhkEw44dV9DecmI+r+FZIh72eFwGx8NZLr0erSkknkUJwS/reZgejG0nGevfqM6Sow8Zat29Xw==}
+    resolution: {integrity: sha1-iIRCw+KXZ33jaIf7MlEAonFS0CU=}
```

Comment:
These need to be changed back to SHA512 -- SHA1 violates policy.

Perhaps you were using an older NPM version?